src/parser.js

```diff
@@ -53,12 +53,34 @@ export function getSliceRange(label) {
     return res ? res.slice(1) : [];
 }
 
-export function embedCode(lang, filePath, originalPath, start, end) {
+/*
+ format: [import:<markername>](path/to/file)
+ marker name start with alpha [a-Z] char.
+
+ Example:
+
+ [import:markname, hello-world.js](../src/hello-world.js)
+ */
+export function getMarkerName(label) {
```

Comment:
It should be moved to `marker.js`.

And split it into `getMarkerName()` and `hasMarker()` function.